tests/test_bar_data.py

```diff
@@ -89,115 +88,100 @@ def check_internal_consistency(self, bar_data):
                 getattr(bar_data, field)
 
 
-class TestMinuteBarData(TestBarDataBase):
+class TestMinuteBarData(WithBarDataChecks,
+                        WithDataPortal,
+                        ZiplineTestCase):
+    SIM_PARAMS_START = pd.Timestamp('2016-01-05', tz='UTC')
+    SIM_PARAMS_END = pd.Timestamp('2016-01-07', tz='UTC')
+
+    SPLIT_ASSET_SID = 3
+    ILLIQUID_SPLIT_ASSET_SID = 4
+    HILARIOUSLY_ILLIQUID_ASSET_SID = 5
+
     @classmethod
-    def setUpClass(cls):
-        cls.tempdir = TempDirectory()
+    def make_equity_info(cls):
+        return pd.DataFrame.from_dict(
+            {
+                sid: {
+                    'start_date': cls.SIM_PARAMS_START,
+                    'end_date': cls.SIM_PARAMS_END,
+                    'symbol': "ASSET{0}".format(sid)
+                } for sid in [1, 2, 3, 4, 5]
+            },
+            orient='index',
+        )
 
+    @classmethod
+    def make_minute_bar_data(cls):
         # asset1 has trades every minute
         # asset2 has trades every 10 minutes
         # split_asset trades every minute
         # illiquid_split_asset trades every 10 minutes
-
-        cls.env = TradingEnvironment()
-
-        cls.days = cls.env.days_in_range(
-            start=pd.Timestamp("2016-01-05", tz='UTC'),
-            end=pd.Timestamp("2016-01-07", tz='UTC')
-        )
-
-        cls.env.write_data(equities_data={
-            sid: {
-                'start_date': cls.days[0],
-                'end_date': cls.days[-1],
-                'symbol': "ASSET{0}".format(sid)
-            } for sid in [1, 2, 3, 4, 5]
-        })
-
-        cls.ASSET1 = cls.env.asset_finder.retrieve_asset(1)
-        cls.ASSET2 = cls.env.asset_finder.retrieve_asset(2)
-        cls.SPLIT_ASSET = cls.env.asset_finder.retrieve_asset(3)
-        cls.ILLIQUID_SPLIT_ASSET = cls.env.asset_finder.retrieve_asset(4)
-        cls.HILARIOUSLY_ILLIQUID_ASSET = cls.env.asset_finder.retrieve_asset(5)
-
-        cls.ASSETS = [cls.ASSET1, cls.ASSET2]
-
-        cls.adjustments_reader = cls.create_adjustments_reader()
-        cls.data_portal = DataPortal(
-            cls.env,
-            equity_minute_reader=cls.build_minute_data(),
-            adjustment_reader=cls.adjustments_reader
+        return merge(
+            {
+                sid: create_minute_df_for_asset(
+                    cls.env,
+                    cls.bcolz_minute_bar_days[0],
+                    cls.bcolz_minute_bar_days[-1],
+                )
+                for sid in (1, cls.SPLIT_ASSET_SID)
```

Comment:
It just seems strange to me to refer to some of the assets symbolically but refer to others by number.  I don't care strongly though.